project-base/app/config/packages/snc_redis.yml

```diff
@@ -5,19 +5,25 @@ snc_redis:
             alias: 'bestselling_products'
             dsn: 'redis://%redis_host%'
             options:
-                prefix: '%env(REDIS_PREFIX)%bestselling_products_'
+                prefix: '%env(REDIS_PREFIX)%%build-version%bestselling_products_'
```

Comment:
would some additional `_` as delimiters make sense or not?

Reply:
I don't see much added value in that